legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/protocols/pure/vX_X_X/models/metamodel_connection.pure

```diff
@@ -84,6 +91,15 @@ Class meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::a
     defaultDataset:String[1];
 }
 
+Class meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::specification::RedshiftDatasourceSpecification extends meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::specification::DatasourceSpecification
+{
+    clusterName : String[1];
+    clusterID : String[1];
+    databaseName : String[1];
+    port : Integer[1];
+    region : meta::pure::alloy::connections::alloy::specification::Region[1];
```

Comment:
Don't think region should be a enum for a couple of regions 

1/ Symmetry - Snowflake regions are modeled as plain Strings 

2/ New regions - Cloud providers are always introducing new regions. It is going to be annoying to release a new version of Legend just because AWS decided to offer a new region

Reply:
@epsstan makes sense. I wonder how frequently are the regions added. I guess enums are better from user-perspective as then can select from a list and avoid typo's
If its rare maybe we can have enum in studio only but if its frequent I guess we are better of with strings

Reply:
I have changed it back to a string in the update

Reply:
> @epsstan makes sense. I wonder how frequently are the regions added. I guess enums are better from user-perspective as then can select from a list and avoid typo's
> If its rare maybe we can have enum in studio only but if its frequent I guess we are better of with strings

Regions are more frequent than you think.  Italy, South Africa, and Japan came online in the last 12 months. Spain is expected in 2022.